openhtf/io/frontend/__main__.py

```diff
@@ -225,40 +214,65 @@ def post(self, host, port, prompt_id):
     msg = json.JSONEncoder().encode(
         {'id': prompt_id, 'response': self.request.body})
     self.store[host, port].Notify(msg)
-    self.write('SENT')
+    self.write('')
 
 
 def main(argv):
   """Start the frontend."""
-  try:
-    argv = FLAGS(argv)  # parse flags
-  except gflags.FlagsError, e:
-    print('%s\nUsage: %s ARGS\n%s' % (e, sys.argv[0], FLAGS))
-    sys.exit(1)
+  parser = argparse.ArgumentParser(description='OpenHTF web frontend server.',
+                                   prog='python -m openhtf.io.frontend')
+  parser.add_argument('--port', type=int, default=12000,
+                      help='Port on which to serve the frontend.')
+  parser.add_argument('--poll_interval_ms', type=int, default=3000,
+                      help='Time between frontend polls in milliseconds.')
+  parser.add_argument('--discovery_interval_s', type=int, default=5,
+                      help='Seconds between station discovery attempts.')
+  parser.add_argument('--discovery_address', type=str,
+                      default=multicast.DEFAULT_ADDRESS,
+                      help='Multicast address to ping for station discovery.')
+  parser.add_argument('--discovery_port', type=int,
+                      default=multicast.DEFAULT_PORT,
+                      help='Multicast port to use for station discovery.')
+  parser.add_argument('--discovery_ttl', type=int,
+                      default=multicast.DEFAULT_TTL,
+                      help='TTL for station discovery pings.')
+  parser.add_argument('--dev', action='store_true',
+                      help='Start in development mode.')
+  args = parser.parse_args()
 
   logs.setup_logger()
 
-  store = StationStore()
+  path = BUILD_PATH if os.path.exists(BUILD_PATH) else PREBUILT_PATH
```

Comment:
I went back and forth a little on this, actually. If you use the flag, you might end up with nothing to serve, because there's nothing forcing you to actually build. On the other hand, if you _do_ build, but you want to serve the prebuilt for comparison or something, there's no simple way to do it. You either have to blow away what you built, or go in and change the path in code. I could go either way on this.